src/main/java/cz/tstrecha/timetracker/service/impl/TaskServiceImpl.java

```diff
@@ -113,4 +116,13 @@ public List<TaskDTO> searchForTasks(String query, Long limit, LoggedUser loggedU
                 .stream().map(taskMapper::toDTO).toList();
     }
 
+    @Override
+    @Transactional
+    public List<TaskDTO> listTasks(TaskFilter taskFilter, LoggedUser loggedUser) {
+        var enumValue = Enum.valueOf(Sort.Direction.class, taskFilter.getSortDirection().name());
```

Comment:
Thats weird line.. Weird naming and why calling generic `Enum.valueOf`, instead of `Sort.Direction.valueOf()`?